aws-android-sdk-core/src/main/java/com/amazonaws/internal/keyvaluestore/KeyNotGeneratedException.java

```diff
@@ -16,8 +16,5 @@ public KeyNotGeneratedException(String s, Throwable throwable) {
     public KeyNotGeneratedException(Throwable throwable) {
         super(throwable);
     }
-
-    protected KeyNotGeneratedException(String s, Throwable throwable, boolean b, boolean b1) {
-        super(s, throwable, b, b1);
```

Comment:
This was not being used, and it has an insanely high API level requirement. Get rid of it!